### ISSUE TITLE
sparqljs import issue

### ISSUE THREAD

**Issue by BenjaminHofstetter** [zazuko/query-rdf-data-cube#49]
I am using /@zazuko/query-rdf-data-cube in an Angular App. The compiler complains about sparqljs like this. 

```bash

    ERROR in node_modules/@zazuko/query-rdf-data-cube/dist/node/query.d.ts:6:44 - error TS2307: Cannot find module './sparqljs'.

    6 import { FilterPattern, SelectQuery } from "./sparqljs";
                                                 ~~~~~~~~~~~~
```
I am not sure but maybe it should be like this...

```typescript
import { FilterPattern, SelectQuery } from "sparqljs";
```

**Comment by vhf** [comment]
The error is in a typings file, not in code. `./sparqljs` refers to this file: https://github.com/zazuko/query-rdf-data-cube/blob/2c8dfbb7273c5fd803922887f9c9a50c83a3ecbe/src/sparqljs.d.ts .

It looks like this file is missing from the npm package, I'll look into it.

**Comment by vhf** [comment]
Published [@zazuko/query-rdf-data-cube@0.7.1](https://www.npmjs.com/package/@zazuko/query-rdf-data-cube/v/0.7.1)
